content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -20,40 +20,51 @@ quotations by navigating to the :guilabel:`Optional Products` tab on a quotation
    :align: center
    :alt: How to add optional products to your quotations on Odoo Sales.
 
+To add an optional product(s) to a quotation, click :guilabel:`Add a product` in the
+:guilabel:`Optional Products` tab of a quotation. Doing so reveals a blank field in the
+:guilabel:`Product` column.
+
+When clicked, a drop-down menu with products from the database appear. Select the desired product
+from the drop-down menu to add it as an optional product to the quotation template.
+
 .. tip::
-   Click the :guilabel:`Customer Preview` button to reveal a preview of the quotation customers
-   would receive via email, along with the optional products they have a chance of adding to their
-   order.
+   If the desired product isn't readily visible, type the name of the desired product in the field,
+   and the option appears in the drop-down menu. Then, select that desired product to add it to the
+   quotation.
 
-   .. image:: optional_products/optional-products-checkout.png
-      :align: center
-      :alt: Preview your quotations on Odoo Sales.
+.. note::
+   When a product is added, the default :guilabel:`Quantity` is `1`, but that can be edited at any
+   time.
 
-In practice, customers are able to add different optional products to an order by clicking the
-:guilabel:`🛒 (shopping cart)` icon, located to the right of the optional product line. If a
-customer selects all the optional products suggested, these additional items are automatically
-filled in on the quotation that is managed by the salesperson.
+To delete any line item from the :guilabel:`Optional Products` tab, click the :guilabel:`🗑️ (trash
+can)` icon.
 
-.. image:: optional_products/optional-products-add-to-cart.png
+Click the :guilabel:`Preview` button, located in the upper-left corner of the quotation, to reveal a
+preview of the quotation customers would receive, via email, along with the optional products they
+can potentially add to their order.
```

Comment:
```suggestion
can potentially add to their order, located in the :guilabel:`Options` section.
```
Admittedly, this new sentence doesn't flow as well as your original, but I found it useful to explicitly say where the optional products are found, in case the viewer can't load the image.